Documentation/how-to-build-WebAssembly.md

```diff
@@ -30,7 +30,7 @@
 # How to debug the IL->WebAssembly compilation #
 This is Windows only for now.
 1. Set the ILCompiler startup command line to ```@C:\corert\tests\src\Simple\HelloWasm\obj\Debug\wasm\native\HelloWasm.ilc.rsp```. That will generate HelloWasm.bc, an LLVM bitcode file.
-2. To compile that to WebAssembly, run ```emcc HelloWasm.bc  -s ALLOW_MEMORY_GROWTH=1  C:\corert\bin\WebAssembly.wasm.Debug\sdk\libPortableRuntime.bc C:\corert\bin\WebAssembly.wasm.Debug\sdk\libbootstrappercpp.bc -s WASM=1 -o HelloWasm.html``` (if emcc isn't on your path, you'll need to launch an Emscripten command prompt to do this). That will generate a .wasm file with your code as well as html and js files to run it.
+2. To compile that to WebAssembly, run ```emcc HelloWasm.bc  -s ALLOW_MEMORY_GROWTH=1  C:\corert\bin\WebAssembly.wasm.Debug\sdk\libPortableRuntime.a C:\corert\bin\WebAssembly.wasm.Debug\sdk\libbootstrappercpp.a -s WASM=1 -o HelloWasm.html``` (if emcc isn't on your path, you'll need to launch an Emscripten command prompt to do this). That will generate a .wasm file with your code as well as html and js files to run it.
```

Comment:
This is correct in that this files are now .a, however the document is out of step in another a couple of other ways also.  The `HelloWasm.bc` is first compiled to a  WebAssembly object file, and then linked as per here.  So instead of 2 steps, there should be 3.  And we need a few more options

2. Compile The bitcode to a WebAssembly object file, run ```emcc.bat HelloWasm.bc -c -o HelloWasm.o -s ERROR_ON_UNDEFINED_SYMBOLS=0 -s DISABLE_EXCEPTION_CATCHING=0 -g``` (if emcc isn't on your path, you'll need to launch an Emscripten command prompt to do this). 
3. To compile that to WebAssembly, run ```emcc HelloWasm.o -o HelloWasm.html -s ERROR_ON_UNDEFINED_SYMBOLS=0 -s DISABLE_EXCEPTION_CATCHING=0 -s WASM_MEM_MAX=200Mb --emrun "c:\corert\bin\WebAssembly.wasm.Debug/sdk/libPortableRuntime.a" "C:\corert\bin\WebAssembly.wasm.Debug\sdk\libbootstrappercpp.a" "C:\corert\bin\WebAssembly.wasm.Debug\sdk\libSystem.Private.CoreLib.Native.a"  -g -s INITIAL_MEMORY=167772160 --js-library C:\corert\tests\src\Simple\HelloWasm\dotnet_support.js --pre-js C:\corert\tests\src\Simple\HelloWasm\Microsoft.JSInterop.js --post-js C:\corert\tests\src\Simple\HelloWasm\HelloWasm.js -g``` That will generate a .wasm file with your code as well as html and js files to run it.